### PR TITLE
Michael shinydevel

### PR DESCRIPTION
**Pull request by michaelwitting**
Implementation of a Shiny app to evaluate results stored in a `MatchedSpectra` object

### CODE REVIEW

File: NAMESPACE [hunk 1]
```diff
@@ -18,6 +18,7 @@ export(matchSpectra)
 export(matches)
 export(pruneTarget)
 export(query)
+export(shinyMatchedSpectra)
```
Comment (jorainer) [inline NAMESPACE:21]: was wondering - should we call that method maybe `validateMatchedSpectra` instead? Because that's what it is actually doing.

**Review by jorainer — CHANGES_REQUESTED** [review]
I think we should maybe rename the function. Otherwise I'm in general OK. We would need unit tests but I can add them later (which also helps me understanding the code better).

**Comment by michaelwitting** [comment]
No idea how to write a unit test for a shiny app, but if you have please go ahead. I will change the name.

**Review by jorainer — APPROVED** [review]
Thanks!

**Pull request merged**